aci/compose.go

```diff
@@ -46,18 +46,51 @@ func newComposeService(ctx store.AciContext) aciComposeService {
 
 func (cs *aciComposeService) Up(ctx context.Context, project *types.Project, detach bool) error {
 	logrus.Debugf("Up on project with name %q", project.Name)
+
+	err := autocreateFileshares(ctx, project)
+	if err != nil {
+		return err
+	}
+
 	groupDefinition, err := convert.ToContainerGroup(ctx, cs.ctx, *project, cs.storageLogin)
+	if err != nil {
+		return err
+	}
+
 	addTag(&groupDefinition, composeContainerTag)
+	return createOrUpdateACIContainers(ctx, cs.ctx, groupDefinition)
+}
 
+func (cs aciComposeService) warnKeepVolumeOnDown(ctx context.Context, projectName string) error {
+	cgClient, err := login.NewContainerGroupsClient(cs.ctx.SubscriptionID)
 	if err != nil {
 		return err
 	}
-	return createOrUpdateACIContainers(ctx, cs.ctx, groupDefinition)
+	cg, err := cgClient.Get(ctx, cs.ctx.ResourceGroup, projectName)
+	if err != nil {
+		return err
+	}
+	if cg.Volumes == nil {
+		return nil
+	}
+	for _, v := range *cg.Volumes {
+		if v.AzureFile == nil || v.AzureFile.StorageAccountName == nil || v.AzureFile.ShareName == nil {
+			continue
+		}
+		fmt.Printf("WARNING: fileshare \"%s/%s\" will NOT be deleted. Use 'docker volume rm' if you want to delete this volume\n",
```

Comment:
We ideally shouldn't be printing from the backend.

Reply:
Yes, ideally I agree... need quite some refactoring to allow this, I don't really want to include this in this PR. We have https://github.com/docker/compose-cli/issues/442 to track this